rbac/management/tasks.py

```diff
@@ -18,10 +18,17 @@
 from __future__ import absolute_import, unicode_literals
 
 from celery import shared_task
+from django.core.management import call_command
 from management.principal.cleaner import clean_tenants_principals
 
 
 @shared_task
 def principal_cleanup():
     """Celery task to clean up principals no longer existing."""
     clean_tenants_principals()
+
+
+@shared_task
+def run_migrations_in_worker():
+    """Celery task to run migrations."""
+    call_command("migrate_schemas")
```

Comment:
Curious, is there a way to run this with --executor=parallel?

Reply:
Not that I'm aware of. That's what I was referring to yesterday where we can't run a subprocess in the celery worker. Worth digging into a bit more, but if we have a smaller list of schemas and a way to monitor them, I think it's less important to run them in parallel.